app/views/site/contact.scala

```diff
@@ -220,48 +220,48 @@ object contact:
             ),
             Leaf(
               "security",
-              "Security vulnerability",
+              securityVul(),
```

Comment:
Translating everything is not a goal, because each new translation creates a lot of work for translators, and some runtime cost for the system.

Typically, we don't want to translate niche messages, especially when the targetted audience is supposed to know english. Like Internet security researchers.